src/services/filters.ts

```diff
@@ -55,3 +63,146 @@ export function isFilterMetadata(filter: { value: string; valueLabels?: string[]
     : filter.value;
   return value === filter.valueLabels?.[0];
 }
+
+/**
+ * Filters json parser prop filters that are not currently used in either the line filters or the field filters
+ * @todo unit test
+ */
+export function filterUnusedJSONParserProps(sceneRef: SceneObject) {
```

Comment:
I see that it's because these are "prop filters", but I wouldn't mind calling them just filters.

Reply:
Fixed: https://github.com/grafana/logs-drilldown/pull/1122/commits/0d5bc1ee87dc1149dd1c9566e684431caa85098c